src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+$darkBlue: #060b35;
+$gray: #616070;
+$mainColor: #00acdc;
+$white: #fff;
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background-color: $white;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
better to define global styles in `body`.

move font-family to the `body`

Reply:
I don't think this is a good idea because using * can have a negative effect on performance. 
[using star (2).txt](https://github.com/mate-academy/layout_product-cards/files/12122237/using.star.2.txt)

Reply:
here I write about the body, as I understand this question related to the comment above.

Can you please describe to me, what you gonna do if you have 30 different blocks on the page, will you write box-sizing to all these blocks? In a big project if somebody forgot to add box-sizing it can be hard to debug in the future. 

Probably you will have the same box-sizing for all your elements so let's use a selector which specialized in this.

what about optimization, does (*) have a negative effect on the performance? I think in some cases it really can be a problem but not with box-sizing.

[Simple Selectors 1,51 min](https://mate.academy/learn/css-basics/css-selectors?section=video&videoId=4330)
"Я вважаю що використовутати зірочку для ресета не варто, оскільки у більшості єлементі не буде padding/marign а браузеру доведется застосувати ці значення абсолютно до всіх єлементів."